src/components/editor/editor.tsx

```diff
@@ -62,22 +62,37 @@ https://vimeo.com/23237102
 
 ## Code highlighting
 \`\`\`javascript=
+
 let a = 1
 \`\`\`
 
 `)
   const isWide = useMedia({ minWidth: 576 })
   const [firstDraw, setFirstDraw] = useState(true)
+  const [documentTitle, setDocumentTile] = useState('Untitled')
```

Comment:
```suggestion
  const [documentTitle, setDocumentTitle] = useState('Untitled')
```

This needs proper renaming through the code as the set-method is used multiple times.

Reply:
ups. fixed that